src/main/java/core/basesyntax/dao/StorageDaoImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+
+public class StorageDaoImpl implements StorageDao {
+
+    @Override
+    public void add(Fruit fruit) {
+        Storage.fruits.put(fruit.getFruitName(), fruit);
+    }
+
+    @Override
+    public Fruit get(String fruitName) {
+        return Storage.fruits.get(fruitName);
+    }
+}
```

Comment:
The `StorageDaoImpl` class directly interacts with the `Storage` class's static field `fruits`. This violates the encapsulation principle. Consider creating methods in the `Storage` class to interact with its data, rather than accessing it directly from other classes.